packages/gatsby/src/bootstrap/index.js

```diff
@@ -72,16 +73,7 @@ module.exports = async (args: BootstrapArgs) => {
   // Try opening the site's gatsby-config.js file.
   activity = report.activityTimer(`open and validate gatsby-config.js`)
   activity.start()
-  let config
-  try {
-    // $FlowFixMe
-    config = preferDefault(require(`${program.directory}/gatsby-config`))
-  } catch (err) {
-    if (!testRequireError(`${program.directory}/gatsby-config`, err)) {
-      report.error(`Could not load gatsby-config`, err)
-      process.exit(1)
-    }
-  }
+  const config = await preferDefault(getConfigFile(program.directory, 'gatsby-config.js'));
```

Comment:
No issues with `await`ing here right? The "happy path" shouldn't see a performance hit I don't think.

Reply:
Nope, this should be fine.